multidict/_multidict_base.py

```diff
@@ -46,6 +46,8 @@ def _viewbaseset_richcmp(view, other, op):
             if elem not in view:
                 return False
         return True
+    else:  # pragma: no cover
+        return NotImplemented
```

Comment:
Why is this needed? Are there any other `op`s that exist?

Should this be `assert_never()` per https://stackoverflow.com/a/74535743/595220?

If there's other `op`s, let's cover this branch by using them in tests.

Reply:
If this is needed, add a bugfix change note. If not, don't include it and add a contrib change note.

https://github.com/aio-libs/multidict/tree/master/CHANGES#adding-change-notes-with-your-prs

Reply:
`op` can only be one of the six values(https://docs.python.org/3/c-api/typeobj.html#c.PyTypeObject.tp_richcompare). I added this so that the coverage doesn't report that the previous `elif` statement isn't fully covered. The previous `elif` should never to false, as these `if/elif` statements covers all legal values.

`assert_never()` only exists in Python 3.11 and above, so have also added a fallback for versions prior to that. 

As this doesn't change any user behavior, should this be a contrib change note, or a bugfix change note?

Reply:
Contrib is good. As you mentioned — this doesn't bring end-user facing changes.

Reply:
I'm marking this thread as resolved, since I made a fallback suggestion in another place.